src/test/java/ch/uzh/ifi/hase/soprafs22/screwyourneighborserver/validation/GameValidatorTest.java

```diff
@@ -12,8 +12,10 @@
 import org.junit.jupiter.params.ParameterizedTest;
 import org.junit.jupiter.params.provider.CsvSource;
 import org.junit.jupiter.params.provider.EnumSource;
+import org.springframework.test.annotation.DirtiesContext;
 import org.springframework.web.client.HttpClientErrorException;
 
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_CLASS)
```

Comment:
?

Reply:
Das räumt die Mocks auf zwischen den Testklassen. Die Testklasse lief nur, wenn ich sie isoliert laufen liess, aber nicht wenn ich alle Tests zusammen laufen liess (z.B. beim Build). Und dann habe ich das gefunden: https://stackoverflow.com/questions/18164123/how-to-clean-up-mocks-in-spring-tests-when-using-mockito und das: https://www.baeldung.com/spring-dirtiescontext
Mit dieser Annotation hat's dann funktioniert.

Reply:
@DirtiesContext tells Spring that it should recreate the ApplicationContext. This is for @MockMvcTest, @SpringBootTest or @DataJpaTest, but not for plain unit tests.
I removed this one and the tests run through on my machine without the annotation. (In fact i removed both @DirtiesContext annotations, and all tests passed).

Reply:
Then why is it still here?

Reply:
Because I'm obviously not able to merge your commit proposals and my changes without losses :angry: